exampleSite/utilities/examples/data/events/yyyy-city.yml

```diff
@@ -46,6 +52,11 @@ team_members: # Name is the only required field for team members.
     twitter: "devopsdays"
   - name: "Sally Fields"
     employer: "Acme Anvil Co."
+    github: "devopsdays"
+    facebook: "https://www.facebook.com/sally.fields"
+    linkedin: "https://www.linkedin.com/in/sallyfields"
+    website: "http://devopsdays.org"
```

Comment:
Probably. It was late and we use "devopsdays" as our default for everything else, but I could make up a site. The problem with a sample is that if they don't change it (although they will, since it's a fake person) it could point to malware site. This happened with hugo where they put in "http://yoursitehere.com" as the default value in a config, and it was spreading malware.